src/Evolution/Systems/GrMhd/ValenciaDivClean/System.hpp

```diff
@@ -35,6 +35,138 @@ namespace grmhd {
 ///
 namespace ValenciaDivClean {
 
+/*!
+ * \brief General relativistic magnetohydrodynamics (GRMHD) system with
+ * divergence cleaning coupled with electron fraction.
+ *
+ * We adopt the standard 3+1 form of metric
+ * \f{align*}
+ *  ds^2 = -\alpha^2 dt^2 + \gamma_{ij}(dx^i + \beta^i dt)(dx^j + \beta^j dt)
+ * \f}
+ * where \f$\alpha\f$ is the lapse, \f$\beta^i\f$ is the shift vector, and
+ * \f$\gamma_{ij}\f$ is the spatial metric.
+ *
+ * Primitive variables of the GRMHD system are
+ *  - rest mass density \f$\rho\f$
+ *  - electron fraction \f$Y_e\f$
+ *  - the spatial velocity \f{align*} v^i =
+ * \frac{1}{\alpha}\left(\frac{u^i}{u^0} + \beta^i\right) \f} measured by the
+ * Eulerian observer
+ *  - specific internal energy density \f$\epsilon\f$
+ *  - magnetic field \f$B^i = -^*F^{ia}n_a = -\alpha (^*F^{0i})\f$
+ *  - the divergence cleaning field \f$\Phi\f$
+ *
+ * with corresponding derived physical quantities which frequently appear in
+ * equations:
+ * - The transport velocity \f$v^i_{tr} = \alpha v^i - \beta^i\f$
+ * - The Lorentz factor \f{align*} W = -u^an_a = \alpha u^0 =
+ * \frac{1}{\sqrt{1-\gamma_{ij}v^iv^j}} = \sqrt{1+\gamma^{ij}u_iu_j} =
+ * \sqrt{1+\gamma^{ij}W^2v_iv_j} \f}
+ * - The specific enthalpy \f$h = 1 + \epsilon + p/\rho\f$ where \f$p\f$ is the
+ * pressure specified by a particular equation of state (EoS)
+ * - The comoving magnetic field
```

Comment:
Maybe say `(in the ideal MHD limit)`, since this is not the the general comoving field expression.
You can also add the general definition, 
`\f$b^b = -^*F^{ba}u_a \f$`